### ISSUE TITLE
Is it possible to use google aiy voice kit to create a translator? 

### ISSUE THREAD

**Issue by ryanbaejh** [google/aiyprojects-raspbian#533]
Like when you talk to them, they talk back to you

**Comment by andrewtvuong** [comment]
yes it should be in terms of hardware. The easiest way I can think of is to utilize google translate API somehow. feed it your voice commands, let it parse it into text, send to google translate, receive translation, spit it back out.

**Comment by dmitriykovalev** [comment]
It's totally possible. Voice Kit gives you the hardware to input/output sound, everything else is software you can install and use.
